### PR TITLE
Main project

### PR DESCRIPTION
**Pull request by arslanbisharat**
This project has the following features:
1-Project considers responsiveness on each Bootstrap breakpoint
2-Most of the styling and positioning is done using Bootstrap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# News-Week.com Clone
+In this project I have build a replica of the news site Newsweek.com using the Bootstrap framework.
+
+## Project features:
+1. Project considers responsiveness on each Bootstrap breakpoint
+2. Most of the styling and position is done using Bootstrap
+
```
Comment (majovanilla) [inline README.md:7]: - Good description :heavy_check_mark:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Microverse Curriculum. 
+
+### Run tests
+
+Text editor and web browser.
+
+### Deployment
+
+Localhost. 
+
```
Comment (majovanilla) [inline README.md]: - [ ] Some of these sections are not necessary for projects like this one. Feel free to delete those you don't need.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+Bitcoin donations accepted ;)
```
Comment (majovanilla) [inline README.md]: :laughing: where's your wallet?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,801 @@
...
+                    <li class="nav-item">
+                        <a class="nav-link px-0" href="#">The Debate</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link px-0" href="#">Vintage</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link px-0" href="#">Weather</a>
+                    </li>
+                </ul>
```
Comment (majovanilla) [inline index.html]: - [ ] Before hiding it, your navbar items should occupy all the space available. Right now they looked aligned to the left.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,801 @@
...
+                <form class="form-inline pl-4" action="/">
+                    <input class="border-0" type="text" placeholder="Search">
+                </form>
```
Comment (majovanilla) [inline index.html]: - [ ] It's ok to use an input without the enclosing `<form>` tag. **optional**

File: index.html [hunk 6]
```diff
@@ -0,0 +1,801 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
+        integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+    <title>News: Newsweek.com</title>
+</head>
+<body>
```
Comment (majovanilla) [inline index.html]: - [ ] Check the body of the original page. It has a gray background color that you can see when the screen is large and then the content starts occupying more space until you have a 100% width.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,801 @@
...
+                        <a class="nav-link px-0" href="#">Weather</a>
+                    </li>
+                </ul>
+                <form class="form-inline pl-4" action="/">
+                    <input class="border-0" type="text" placeholder="Search">
+                </form>
+            </nav>
+        </header>
+
+        <main class="container-fluid p-md-4 pb-0">
```
Comment (majovanilla) [inline index.html]: - [ ] You can achieve the same look on the page by adding some `col-*` with responsiveness to the `<main>` in addition to the `container-fluid` class.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,801 @@
...
+                    </section>
+
+                    <section class="latest-news mt-5">
+                        <h2 class="text-secondary font-weight-bold">LATEST NEWS</h2>
+                        <article class="row">
+                            <img class="col-lx-4 col-lg-5 col-md-2 col-3 mr-3 mx-md-0 pr-md-4"
+                                src="https://d.newsweek.com/en/full/1573026/one-chicago-series-next.webp?w=95&h=95&f=e4f6ef4ad72a5d74b50840e7b1214898"
+                                alt="">
+                            <h5 class="col pl-0 mt-1 pr-4 mt-md-0 row font-weight-bold">Why New Episodes of 'Chicago Fire,' 'Hope' and 'PD' Aren't Airing This Week</h5>
+                        </article>
```
Comment (majovanilla) [inline index.html]: - [ ] The articles' images are larger than the original page. This happens in most breakpoints. Please refer to the Newsweek page and make the necessary adjustments to make your site look as close as theirs.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,801 @@
...
+                    <section class="more-stories order-2 my-4">
+                        <h2 class="text-secondary font-weight-bold my-2">MORE STORIES</h2>
+                        <article class="row">
+                            <img class="col-3"
+                                src="https://d.newsweek.com/en/full/1572711/joe-biden-detroit-michigan.webp?w=95&h=95&f=8b682ca8c8fde4bfb26dc613809858dc"
+                                alt="">
+                            <div class="col-9 pl-0">
+                                <h5 class="font-weight-bold">Joe Biden Beats Bernie Sanders by Double Digits in Eight Michigan Polls</h5>
+                                <p class="m-0">A ninth poll conducted by YouGov put the former vice president 8 points ahead of his progressive
+                                    challenger ahead of Tuesday's contests.</p>
```
Comment (majovanilla) [inline index.html]: - [ ]  The articles' images are larger than the original page. This happens in most breakpoints. Please refer to the Newsweek page and make the necessary adjustments to make your site look as close as theirs.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,801 @@
...
+                            <img class="category-aside-img col-lg-5 col-md-6 col-3 px-3 pl-4"
+                                src="https://d.newsweek.com/en/full/1572992/sanitizer.webp?w=95&h=95&f=04ec8177c12a5cc2221e8c18388de99a"
+                                alt="">
+                            <h5 class="col px-0 font-weight-bold">7-Eleven Owner Accused of Selling Homemade Sanitizer Which Burned Children</h5>
+                        </article>
+                    </aside>
+                </div>
+            </section>
+
+            <div class="subscribe-and-newsletters-bot row border-top border-secondary pt-5 py-md-5 my-3">
```
Comment (majovanilla) [inline index.html]: - [ ] If you check the original page, you will see that the divisor you are using before this section is not a border, but the actual gray background that the body has. They achieve this look by giving some margin. Please make the necessary changes to achieve the same effect on your project.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,801 @@
...
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Active Volcanoes Around the World That Could Erupt at Any Moment</h3>
+                    </article>
+                </div>
+            </section>
+            <hr>
+
+            <section class="category py-3">
+                <h3 class="category-title">U.S.</h3>
+                <div class="category-container row">
```
Comment (majovanilla) [inline index.html]: - [ ] You are not showing a lot of the categories after the U.S. Please include all of them on your page. Remember it has to be an exact clone. _Tip:_ you already have the look and behavior, so copy, paste and edit what you need to.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,801 @@
...
+                                src="https://d.newsweek.com/en/full/1573013/seismic-graph.webp?w=95&h=95&f=67ec5335f1746d4f8673ccd72d1a8fd0"
+                                alt="">
+                            <h5 class="col pr-0 pl-2 font-weight-bold">New York Earthquake: South Glens Falls Struck by 3.1 Shake</h5>
+                        </article>
+                        <article class="row">
+                            <img class="category-aside-img col-lg-5 col-md-6 col-3 px-3 pl-4"
+                                src="https://d.newsweek.com/en/full/1572992/sanitizer.webp?w=95&h=95&f=04ec8177c12a5cc2221e8c18388de99a"
+                                alt="">
+                            <h5 class="col px-0 font-weight-bold">7-Eleven Owner Accused of Selling Homemade Sanitizer Which Burned Children</h5>
+                        </article>
```
Comment (majovanilla) [inline index.html]: - [ ] The last article of the aside shouldn't have a border-bottom

File: index.html [hunk 13]
```diff
@@ -0,0 +1,801 @@
...
+                    </article>
+                    <article class="col-md-4">
+                        <img class="category-main-img" src="https://d.newsweek.com/en/full/1573102/super-tuesday-primary-voters.webp?w=397&h=265&f=b7119d3c9cce244963cbbb96b607d0cd"
+                            alt="featured-stories">
+                        <h3 class="pt-2 pt-md-0">Democratic Delegate Count Update: Results For Biden, Sanders and Gabbard</h3>
+                    </article>
+                    <aside class="category-aside col-md-4 row">
+                        <article class="row">
+                            <img class="category-aside-img col-lg-5 col-md-6 col-3 px-3 pl-4"
+                                src="https://d.newsweek.com/en/full/1573028/dr-anthony-fauci-covid-19-briefing.webp?w=95&h=95&f=2e43e6e9e762cda94d8010ffc0f163af"
```
Comment (majovanilla) [inline index.html]: - [ ] The images on the `aside` of these sections (the ones you are not showing yet) should be a little smaller. Please check all the breakpoints and compare them to the original page.

- [ ] Check the second article's image in the mobile version: it has less width than the others but that should be the size of all the images in that screen size.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,801 @@
...
+            <hr>
+
+            <section class="category py-3">
+                <h3 class="category-title">U.S.</h3>
+                <div class="category-container row">
+                    <article class="col-md-4">
+                        <img class="category-main-img" src="https://d.newsweek.com/en/full/1573074/coronavirus-recoveries-us-global-cases.webp?w=397&h=265&f=0e215f9d6b052a4451c36c89b8126425"
+                            alt="featured-stories">
+                        <h3 class="pt-2 pt-md-0">Over 66,000 Have Recovered From Coronavirus, Per Johns Hopkins University</h3>
+                    </article>
```
Comment (majovanilla) [inline index.html]: - [ ] The images on the main articles of these sections (the ones you are not showing yet) should be a little smaller in the medium and small breakpoints.

File: style.css [hunk 15]
```diff
@@ -0,0 +1,403 @@
...
+  padding-right: 1px;
+  margin: 0 6px;
+  background-color: #e2e2e2;
+}
+
+header input {
+  background-image: url("./imgs/search-icon.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 20px;
```
Comment (majovanilla) [inline style.css]: - [ ] You could use a [background shorthand](https://developer.mozilla.org/en/docs/Web/CSS/background)

File: style.css [hunk 16]
```diff
@@ -0,0 +1,403 @@
...
+.magazines-container img {
+  width: 100%;
+  height: 60%;
+  object-fit: cover;
+}
+
+@media only screen and (max-width: 768px) {
+  .magazines-container img {
+    height: 78%;
+  }
```
Comment (majovanilla) [inline style.css]: - [ ] You repeat your media queries a lot. That is not a good practice. Please move all the elements inside the appropriate media query and delete the repeated ones.

File: style.css [hunk 17]
```diff
@@ -0,0 +1,403 @@
...
+h5.subscribe-and-newsletters-title {
+  font-size: 16px;
+}
+
+.newsletter-bot-right,
+.subscribe-right {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
```
Comment (majovanilla) [inline style.css]: - [ ] Please use bootstrap classes.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey @arslanbisharat , 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯
- Your page is an almost perfect clone!
- Good Readme file
- Good Pull Request Message
- Great attention to detail 

:star: I can tell you made an effort on this project and you should be proud! 

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- [ ] Check your linters. I cannot see them working!!

- [ ]  You are using a lot of width properties in your classes. You should achieve the same effect using the `col-*` bootstrap classes.

- [ ] Check the comments below and make the necessary adjustments.

- [ ] There is no need to include media queries. You should achieve the same look using bootstrap (they already did all of the work!!) **optional**

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by lekegitrepo** [comment]
### Hi @arslanbisharat 👏 👏 
### _You have done a great job by implementing the required changes and the page looks great now but you need to make some adjustments on the page before this project can be approved._

**_1. At 768px and below, this part of the page stack upon each other 👇_** 

image

**_2. Try to make the navbar fit the with of the page at each breakpoint_**

newsweek

**_Implement the required changes and submit for review_**

### _Reviewed by: Temitatyo Adeleke_

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
### Status: Require changes

**Comment by thsvr** [comment]
### **STATUS: PROJECT APPROVED** :fireworks: :fireworks:  :fireworks:  :fireworks: 

Hi @arslanbisharat  :smile: You did all the required adjustments and your project looks amazing :1st_place_medal: :1st_place_medal: Congratulations on all your hard work and dedication!

**Optional Suggestions:**

:v: On your screenshot, on the readme file,  there are 2 white gaps on the left and right side below the navbar. Can you please replace it with a new print? Your page looks perfect and the screenshot doesn't show it properly.


If you have any questions, don't hesitate to contact me through slack [@Thaís Vieira Azevedo](url)

### **Happy coding and good luck on the next project** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Pull request merged**